charts/instana-agent/v1.2.1/Chart.yaml

```diff
@@ -1,5 +1,5 @@
-apiVersion: v1
-appVersion: 1.1
+apiVersion: v2
```

Comment:
This repository is for Helm 2 charts, so this change will cause the installation to fail in Rancher. Your options would be to either keep the Helm 2 compatibility, or move the chart to [rancher/helm3-charts](https://github.com/rancher/helm3-charts/) and update the `app-readme.me` in the previous version to let users know new versions will be in the Helm 3 catalog. If you decide to move it, all you have to do is move the commit to that repo and submit a PR. Basically the same you did here, but in that other repo.